data/mil2525d/utilities/military-features-utilities/README.md

```diff
@@ -20,23 +23,25 @@
 
 ## Instructions
 
-### Create Military Features Geodatabase Template
+### Create Military Feature Geodatabase Template
```

Comment:
Lorraine was quite insistent that this was plural "features" - but this change is OK - I think we are going to rename all of the "military feature(s)" references in this new Pro solution anyway - to better distinguish from the previous military features solution. Picking a new name is on our TODO list.